### PR TITLE
Add test for base64 encoded images

### PR DESCRIPTION
**Pull request by marckrenn**

### CODE REVIEW

File: test/tests/LayerTest.coffee [hunk 1]
```diff
@@ -256,6 +256,25 @@ describe "Layer", ->
 			#layer.computedStyle()["background-size"].should.equal "cover"
 			#layer.computedStyle()["background-repeat"].should.equal "no-repeat"
 
+					
+		it "should not append nocache to a base64 encoded image", ->
+
+			prefix = "../"
```
Comment (eelco) [inline test/tests/LayerTest.coffee:262]: I think the prefix should be dropped, that does not make sense for a data image.

File: test/tests/LayerTest.coffee [hunk 2]
```diff
@@ -256,6 +256,25 @@ describe "Layer", ->
...
+			imagePath = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEwAAABMBAMAAAA1uUwYAAAAAXNSR0IArs4c6QAAABVQTFRFKK/6LFj/g9n/lNf9lqz/wez/////Ke+vpgAAAK1JREFUSMft1sENhDAMBdFIrmBboAjuaWFrsNN/CRwAgUPsTAH556c5WVFKQyuLLYbZf/MLmDHW5yJmjHW5kBljPhczY8zlEmaMvXMZM8ZeuZQZY08uZzZh5dqen+XNhLFBbsiEsW9uzISxTy5gwlifi5gw1uVCJoz5XMyEMZdLmASs/s5NnkFl7M7NmDJ25aZMGTtzc6aMtcqYMtYqY8pYq4wpY60ypuvnsNizA+E6656TNMZlAAAAAElFTkSuQmCC"
```
Comment (eelco) [inline test/tests/LayerTest.coffee:263]: Nice image ;)

**Review by eelco — CHANGES_REQUESTED** [review]
One tiny thing that I’d change. (I don’t mean to be harsh about it, mostly just playing with Githubs new review feature ;)

File: test/tests/LayerTest.coffee (hunk 2 above)
Comment (jordandobson) [reply test/tests/LayerTest.coffee:263]: Heh. I added a few others for testing as well. 💃

File: test/tests/LayerTest.coffee (hunk 1 above)
Comment (jordandobson) [reply test/tests/LayerTest.coffee:262]: Totally... I removed it in following commits.

Comment (marckrenn) [reply test/tests/LayerTest.coffee:262]: In my defense, I wasn't 100% sure of the prefix's role in the context of a test, so I decided to simply keep it :)

Sorry!

**Pull request merged**